libs/libc/tls/Kconfig

```diff
@@ -54,4 +54,14 @@ config TLS_NELEM
 		NOTE that the special value of CONFIG_TLS_NELEM disables these
 		TLS interfaces.
 
+config TLS_TASK_NELEM
+	int "Number of Task Local Storage elements"
+	default 0
```

Comment:
I think that together with ``uintptr_t ta_telem[CONFIG_TLS_TASK_NELEM]; it breaks C89 compatibility

Reply:
There is a macro guard `#if CONFIG_TLS_TASK_NELEM > 0` for it.

Reply:
Sorry. Missed that!